src/etc/lldb_batchmode.py

```diff
@@ -124,7 +124,7 @@ def listen():
                         breakpoint = lldb.SBBreakpoint.GetBreakpointFromEvent(event)
                         print_debug("breakpoint added, id = " + str(breakpoint.id))
                         new_breakpoints.append(breakpoint.id)
-        except:
+        except Exception:
```

Comment:
This kind of change is potentially behaviour-breaking, e.g. now it won't catch things like `KeyboardInterrupt` (SIGINT). Not sure if it's important in the few places that are changed like this in this PR.

Reply:
That is usually what this sort of change is meant to avoid, but you're right that it could be intentional here. I updated to `BaseException` which will also catch interrupt signals.

Reply:
Did the same thing at the other place, https://github.com/rust-lang/rust/pull/112499/files#diff-da865eb2c4cf9a5b0da807986a9009710610af73f8488e6f8291f057b8592e32R443